### DISCUSSION TITLE
天翼云盘 之间是可以 秒传的吗？

### DISCUSSION THREAD

**Discussion by One-sixth** [AlistGo/alist#6675] [Q&A]
用的版本是 v3.35.0 和 天翼云盘客户端驱动
天翼云盘A和天翼云盘B 都打开了秒传选项
但是 我从一个天翼云盘A复制到天翼云盘B里，看到却是先下载，再上传过去的

**Comment by GitCourser** [comment]
~~理解错误吧
秒传的意思是上传本机文件时,服务器检测到有相同的文件就能秒传,和天翼云官方网站功能一样~~

抱歉, 是我搞错了, 文档里是说天翼可以秒传到百度和天翼, 没用过不知效果,
但文档里本地秒传天翼居然是❌, 但我试过本地是能秒传天翼的, 可能文档没更新吧.
实在不行反正都是天翼, 就直接分享吧
